shared-components/tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["**/*.ts", "**/*.tsx"]
+  "basePath": ".",
```

Comment:
if you do this you should be able to get rid of the relative paths

Reply:
Yes, that is the plan, but i still have problems with the VSCode config